### PR TITLE
Using Bootstrap

### PR DESCRIPTION
**Pull request by rida-elbahtouri**
<h1>Using Bootstrap</h1>

#### What does this PR do?
created a Newsweek website clone based on Bootstrap 4

### in this project I :

- created a new HTML document.

- Think about all the elements on the page and how they are grouped together.

- Add Bootstrap to the project

- Layout the basic structure of the page using empty semantic HTML elements that are appropriately sized and which are positioned by using the appropriate Bootstrap classes.

- Set up the top title and navigation bar.

#### How should this be manually tested?
just clone the project and open the index.html file

### CODE REVIEW

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @rida-elbahtouri, great job with this project :+1: Excellent commit messages :100: PR professional, very descriptive. Great job :slightly_smiling_face: Great use of bootstrap classes. Let us improve our project and make it even better. Kindly address the comments below, so we can approve the project.
- Kindly visit stickler-ci.com so we can enable stickler to check for CSS file
- In the ```Nav``` we are missing the ```current date``` and the main navigations links
``` Clone ```
nav
``` Source ```
nav1
- We are currently missing the ``` latest news section ``` and ``` the featured slideshow ```
ln
fet
- Kindly fix the broken live link in the README file

Kindly make these changes and submit another code review :+1: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Comment by rida-elbahtouri** [comment]
Thank @addod19 for your review, I have worked nav bar as required.

About the *Latest News Section* and *Featured Slideshows section* I am sorry there is no such section in the original page to clone. If there's one, please kindly leave me a screenshot.

The readme file live demo page seems to work very well on my end on the current feature branch: https://rawcdn.githack.com/rida-elbahtouri/-Bootstrap-project/86797a287a47ed33b08b2bdf224ea1f931e1331b/index.html
**other** than that I added  the navbar links and  the Date

File: index.html [hunk 1]
```diff
@@ -0,0 +1,2294 @@
...
+      crossorigin="anonymous"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/css/all.css"
+    />
+    <link rel="stylesheet" href="assets/css/style.css" />
+  </head>
+  <body>
+    <nav class="navbar fixed-top red-back">
```
Comment (lucasmazo32) [inline index.html]: - [ ] Kindly fix the navbar for large screens. At least in my end, it has differences. I also notice that the link provided on the PR is from a previous commit, kindly update it.

Source | Clone
-- | --
image | image

**Review by lucasmazo32 — CHANGES_REQUESTED** [review]
# Changes required :wrench:

You are doing a great job so far, and the project is looking good. 

For the project to get approved, kindly implement the changes mentioned below.

### Happy coding! :computer:

## Lucas Mazo

**Comment by rida-elbahtouri** [comment]
fixed the navbar and updated the link:
https://rawcdn.githack.com/rida-elbahtouri/-Bootstrap-project/7d49d470a04f470c86503638e75027c5a80f564a/index.html

**Comment by peterrobert** [comment]
**pull request under review by peter robert**

File: index.html [hunk 2]
```diff
@@ -0,0 +1,2320 @@
...
+              <p>
+                All Access + Weekly Delivery. Choose A Membership That's Perfect
+                for You!
+              </p>
+              <button
+                class="btn red-back px-2 py-1 text-uppercase text-light font-weight-bold"
+              >
+                Subscribe >
+              </button>
+            </div>
```
Comment (peterrobert) [inline index.html]: **[REQUIRED]** kindly fix the text in this block. It is below the image and hence difficult to read on mobile and medium view
snapshot-1591175528832

**Review by peterrobert — CHANGES_REQUESTED** [review]
**CHANGES REQUIRED** :recycle: 
A very great project so far :clap: Very impressive work with implementing the changes required :+1: 

However, there is one minor change that you should do in order to have this project approved. Kindly do that one change and resubmit the project for review.

Feel free to reach me on slack on @peterrobert  if you need more clarification :smiley:

**Review by peterrobert — APPROVED** [review]
**APPROVED** :clap: :clap: 
Great and impressive work :1st_place_medal: You have done all the requested changes and hence i approve this project :+1:  :100: 

Remember to merge your branch and submit the project completion form :smiley: 
On to the next project. Happy coding :computer:

**Pull request merged**